src/NewCard/primitives/Wrapper.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import styled, { css } from "styled-components";
+
+import mq from "../../utils/mediaQuery";
+import { CardElement } from "./CardElement";
+import defaultTheme from "../../defaultTheme";
+import { getBorder, getBorderRadius } from "./borders";
+
+const topBorderRadius = css`
+  border-top-left-radius: ${getBorderRadius};
+  border-top-right-radius: ${getBorderRadius};
+`;
+
+const bottomBorderRadius = css`
+  border-bottom-left-radius: ${getBorderRadius};
+  border-bottom-right-radius: ${getBorderRadius};
+`;
+
+const Wrapper = styled.div`
+  padding: ${({ theme, noPadding }) => !noPadding && theme.orbit.spaceMedium};
+  ${CardElement};
+  border-bottom: ${({ bottomBorder }) => bottomBorder && getBorder};
+  transition: margin ${({ theme }) => theme.orbit.durationFast} ease-in-out;
+
+  ${({ expanded }) =>
+    expanded &&
+    css`
+      margin: ${({ theme }) => theme.orbit.spaceXSmall} 0;
+      border-radius: ${getBorderRadius};
+    `};
+
+  ${({ roundedTop }) => roundedTop && topBorderRadius}
+
+  ${({ roundedBottom }) =>
+    roundedBottom &&
+    css`
+      border-bottom: ${getBorder};
+      ${bottomBorderRadius}
+    `};
+
+  &:first-of-type {
+    ${topBorderRadius};
+  }
+
+  &:last-of-type {
+    ${bottomBorderRadius};
+    border-bottom: ${getBorder};
+  }
+
+  ${mq.desktop(css`
+    padding: ${({ theme, noPadding }) => !noPadding && theme.orbit.spaceLarge};
+  `)}
+`;
+
+Wrapper.defaultProps = {
+  theme: defaultTheme,
+};
+
+export default Wrapper;
```

Comment:
I would export React component, not styled, so consumers won't have possibility to attach additional and not wanted styles to it in their implementation. Also with this change, I would have `components` folder, not `primitives`.